packages/components/src/components.d.ts

```diff
@@ -1698,6 +1702,10 @@ declare namespace LocalJSX {
     };
     }
     interface MiLocationInfo {
+        /**
+          * @description Level info.
```

Comment:
Run build scrips (or whatever) to make these autogenerated files reflect that is written in the tsx file.